### DISCUSSION TITLE
请问图片无法展示，如何处理？

### DISCUSSION THREAD

**Discussion by cfuzhao** [zai-org/ChatGLM3#527] [Q&A]
比如，在Code Interpreter画个爱心，只有代码，图片展示不出来，这个问题如何解决？

**Comment by cfuzhao** [comment]
如图，代码放到Jupyter Notebook中可以运行并展示，当在chatglm窗口不展示：
![chatglm3](https://github.com/THUDM/ChatGLM3/assets/11405505/06608f54-d253-45aa-9c48-c77457993518)

然后重装ChatGLM3，依然不展示图片，但有时会提供一个无效的链接：
![chatglm3-tool1](https://github.com/THUDM/ChatGLM3/assets/11405505/ec5e3e50-bbbc-4173-b05f-bb5096615bb1)

环境是安装在阿里云PIA/DSW下的，只对[requirements.txt]里的torch版本号做了调整
protobuf
transformers==4.30.2
cpm_kernels
torch==2.0.0
gradio==3.39
sentencepiece
accelerate
sse-starlette
streamlit>=1.24.0
fastapi>=0.104.1
uvicorn~=0.24.0
loguru~=0.7.2
latex2mathml

**Reply by smilesy115** [reply]
我也是这样，请问解决了吗
